docker-compose.yml

```diff
@@ -19,6 +19,8 @@ services:
       - "5000:5000"
     networks:
       - docker_elk
+    depends_on:
+      - elasticsearch
```

Comment:
I've just tested, the whole stack actually works even without the `depends_on` directive :) For a matter of clarity, we'll keep these directive though.